hack/make/.integration-test-helpers

```diff
@@ -67,12 +67,12 @@ run_test_integration() {
 }
 
 run_test_integration_suites() {
+	go get -u github.com/jstemmer/go-junit-report
```

Comment:
we already have gotestsum installed, let's use that: https://github.com/moby/moby/blob/4e83c90ae8362a8205e3728e8fcb655a713a0d87/hack/dockerfile/install/gotestsum.installer